serverless/transaction-manager/src/utils.rs

```diff
@@ -18,6 +18,7 @@ pub(crate) fn parse_send_error(error: String) -> TxnManagerSendError {
     if error_lowercase.contains("out of gas")
         || (error_lowercase.contains("transaction requires at least")
```

Comment:
btw, I noticed this, this error occurs when, when gas account has less ETHs than required, am I right? If yes, then we can remove it from here and create another error type.

Reply:
Updated to a new type